Shared/Supporting Files/Views/WebView.swift

```diff
@@ -45,15 +45,14 @@ struct WebView: UIViewRepresentable {
             switch navigationAction.navigationType {
             case .linkActivated:
                 if let url = navigationAction.request.url {
-                    await Self.openURL(url)
+                    Self.openURL(url)
```

Comment:
Can the body of `openURL(_:)` just be inlined here now? If I remember right, this method was added purely to address the issue of needing to call the `UIApplication` APIs on the main thread.